src/main/java/com/conveyal/gtfs/loader/JdbcGtfsSnapshotter.java

```diff
@@ -49,10 +56,13 @@
     /**
      * @param feedId namespace (schema) to snapshot. If null, a blank snapshot will be created.
      * @param dataSource the JDBC data source with database connection details
+     * @param normalizeStopTimes whether to keep stop sequence values intact or normalize to be zero-based and
+     *                           incrementing
      */
-    public JdbcGtfsSnapshotter(String feedId, DataSource dataSource) {
+    public JdbcGtfsSnapshotter(String feedId, DataSource dataSource, boolean normalizeStopTimes) {
```

Comment:
Should this new parameter be renamed `normalizeStopTimesSequence` or something similar? that's what this parameter utlimately controls in the normalizeAndCloneStopTimes method.

Reply:
I think `normalizeStopTimes` is descriptive enough with good javadoc. The language 'normalize stop times' is used a number of times here to mean a single, well-documented thing.